### PR TITLE
Fix #11931: CSV Bean validation translations

### LINKED ISSUE #11931: CSV: Missing some bean validation keys
For example `@NotBlank` and `@NotEmpty` render like this...

image

### PR DESCRIPTION
**Pull request by melloware**
Fix #11931: CSV NotBlank and NotEmpty translations

For the most part I copied the NotNull value for NotBlank and NotEmpty if anyone wants their locale upated just paste the translation here @jepsar @christophs78 @Rapster

### CODE REVIEW

File: primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-de.js [hunk 1]
```diff
@@ -239,6 +239,8 @@ if (window.PrimeFaces) {
       "javax.validation.constraints.Future.message": "Muss in der Zukunft liegen",
       "javax.validation.constraints.Max.message": "Muss kleiner oder gleich {0} sein",
       "javax.validation.constraints.Min.message": "Muss größer oder gleich {0} sein",
+      "javax.validation.constraints.NotBlank.message": "darf nicht Null sein",
```
Comment (tandraschko) [inline primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-de.js]: both should be
"darf nicht leer sein"

**Comment by NicolaIsotta** [comment]
I just checked Hibernate Validator 8 messages and there are some new keys:
```
jakarta.validation.constraints.Email.message           = must be a well-formed email address
jakarta.validation.constraints.FutureOrPresent.message = must be a date in the present or in the future
jakarta.validation.constraints.Negative.message        = must be less than 0
jakarta.validation.constraints.NegativeOrZero.message  = must be less than or equal to 0
jakarta.validation.constraints.PastOrPresent.message   = must be a date in the past or in the present
jakarta.validation.constraints.Positive.message        = must be greater than 0
jakarta.validation.constraints.PositiveOrZero.message  = must be greater than or equal to 0
```

**Comment by melloware** [comment]
@NicolaIsotta i guess i have to test these to see if they are supportde by CSV.  I know `Email` is not i tried it.

**Comment by melloware** [comment]
OK translations updated for missing bean validation keys.

**Comment by melloware** [comment]
If anyone has any other translations just submit separate PRs....

File: primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js [hunk 2]
```diff
@@ -229,12 +230,34 @@ if (window.PrimeFaces) {
       "javax.faces.validator.LongRangeValidator.MINIMUM": "{1} : Erreur de validation : la valeur est inf\u00e9rieure au minimum autoris\u00e9 de '{0}'",
       "javax.faces.validator.LongRangeValidator.NOT_IN_RANGE": "{2} : Erreur de validation : l'attribut sp\u00e9cifi\u00e9 n'est pas compris entre les valeurs attendues de {0} et de {1}.",
       "javax.faces.validator.LongRangeValidator.TYPE": "{0} : Erreur de validation : la valeur n'est pas du bon type.",
+      "javax.faces.validator.NOT_IN_RANGE": "Erreur de validation : l'attribut sp\u00e9cifi\u00e9 n'est pas compris entre les valeurs attendues de {0} et de {1}.",
       "javax.faces.validator.RegexValidator.NOT_MATCHED": "{1} : Erreur de validation : la valeur ne correspond pas au mod\u00e8le '{0}'",
-      "javax.faces.validator.BeanValidator.MESSAGE": "{1} : {0}",
-      "primefaces.FileValidator.FILE_LIMIT": "Nombre maximum de fichiers dépassé.",
-      "primefaces.FileValidator.FILE_LIMIT_detail": "Nombre maximum : {0}.",
+      "javax.validation.constraints.AssertFalse.message": "ça doit être faux",
+      "javax.validation.constraints.AssertTrue.message": "ça doit être vrai",
```
Comment (Rapster) [inline primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js:236]: doit être faux / doit être vrai

File: primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js [hunk 3]
```diff
@@ -229,12 +230,34 @@ if (window.PrimeFaces) {
...
+      "javax.validation.constraints.DecimalMax.message": "doit être inférieur ou égal à {0}",
+      "javax.validation.constraints.DecimalMin.message": "doit être supérieur ou égal à {0}",
+      "javax.validation.constraints.Digits.message": "valeur numérique hors limites (&lt;{0} chiffres&gt;.&lt;{1} chiffres&gt; attendus)",
+      "javax.validation.constraints.Email.message": "doit être une adresse e-mail bien formée",
+      "javax.validation.constraints.Future.message": "ça doit être une date future",
```
Comment (Rapster) [inline primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js:241]: doit être une date future

File: primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js [hunk 4]
```diff
@@ -229,12 +230,34 @@ if (window.PrimeFaces) {
...
+      "javax.validation.constraints.FutureOrPresent.message": "doit être une date dans le présent ou dans le futur",
+      "javax.validation.constraints.Max.message": "doit être inférieur ou égal à {0}",
+      "javax.validation.constraints.Min.message": "doit être supérieur ou égal à {0}",
+      "javax.validation.constraints.Negative.message": "doit être inférieur à 0",
+      "javax.validation.constraints.NegativeOrZero.message": "doit être inférieur ou égal à 0",
+      "javax.validation.constraints.NotBlank.message": "ne doit pas être vide",
+      "javax.validation.constraints.NotEmpty.message": "ne doit pas être vide",
+      "javax.validation.constraints.NotNull.message": "ne doit pas être nul",
+      "javax.validation.constraints.Null.message": "doit être nul",
+      "javax.validation.constraints.Past.message": "ça doit être une date passée",
```
Comment (Rapster) [inline primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-fr.js:251]: doit être une date passée

**Comment by Rapster** [comment]
Ah just saw your message, I'll comit corrections whenever I can

**Comment by melloware** [comment]
Thanks!

**Pull request merged**